lib/vanagon/platform/dsl.rb

```diff
@@ -85,6 +85,13 @@ def respond_to_missing?(method_name, include_private = false)
         method_name.to_s.start_with?('get_') || super
       end
 
+      def inherit_from_default(name = @name)
```

Comment:
Question: is there a use case where overriding `@name` is useful? Could we get away with this?:

```ruby
def inherit_from_default
  default_file = File.join(__dir__, "/defaults/#{@name}.rb")
  [etc ...]
```

Reply:
I can't think of one, but I left it like that for flexibility. It'd be easy to modify it to your suggestion.

Reply:
Yea, let's. I'm trying to gently nudge us more towards a prescriptive approach inside the RE codebase, removing flexibility as much as is sensible.

It's a tough call between trying to fend off unpredictability/weird corner cases and forbidding valid use.